cmd/update-main.go

```diff
@@ -100,7 +100,7 @@ func GetCurrentReleaseTime() (releaseTime time.Time, err error) {
 
 func isDocker(cgroupFile string) (bool, error) {
 	cgroup, err := ioutil.ReadFile(cgroupFile)
-	if os.IsNotExist(err) {
+	if os.IsNotExist(err) || os.IsPermission(err) {
```

Comment:
I even took the liberty of changing the `docker` check to use `.dockerenv` instead of reading `/proc/cgroup` file. Let me know what you think @balamurugana